docs/networking.adoc

```diff
@@ -126,87 +126,55 @@ We talk about a few major networks in a single-rack Oxide system:
 
 * the **underlay network**.  This is the main network used by Omicron components like Nexus, internal/external DNS, CockroachDB, Sled Agents, Dendrite, and Management Gateway Service to talk to each other.  (See <<rfd61>>.)  Customer traffic is also encapsulated and sent between components on this network.
 * the **bootstrap** network.  This network is used by bootstrap agents (part of the sled agent) to reach other bootstrap agents so that sleds can discover each other and form a trust quorum.  Once the trust quorum is established, sleds can unlock encrypted local storage and start the rest of the control plane, which enables the underlay network to be configured.
-* the **management network**.  This is the network that service processors (SPs) are connected to.  It's constrained, in part because the devices on it have limited capabilities (e.g., limited throughput, generally no TCP, etc.) and also because those devices are critical to the safety of the system (like keeping sleds from melting).  The only control plane component on this network is the Management Gateway Service (MGS).  See <<rfd210>> for more on MGS and the management network.
+* the **management network**.  This is the network that service processors (SPs) and the technician ports on the Sidecars are connected to.  It's constrained, in part because the devices on it have limited capabilities (e.g., limited throughput, generally no TCP, etc.) and also because those devices are critical to the safety of the system (like keeping sleds from melting).  The only control plane component on this network is the Management Gateway Service (MGS).  See <<rfd210>> for more on MGS and the management network.
 * the **external network** or **customer network**.  The Oxide system exposes services directly to customers: namely, external DNS (which serves as a directory for services provided by the rack) and the external API/web console.  The rack's NTP servers (not for external consumption) also connect to other NTP servers using this network.  All of these services use addresses on the customer's network.  This is the only network that can be IPv4.
 
-TODO this should be a diagram
-
-.Summary of what components are on what networks
-[cols="1,1,1,1,1",options="header"]
-|===
-|Component
-|Bootstrap?
-|Management?
-|Underlay?
-|External?
-
-|Sled Agent
-|Yes
-|No
-|No
-|No
-
-|Management Gateway Service (MGS)
-|No
-|Yes
-|Yes
-|No
-
-|Nexus
-|No
-|No
-|Yes
-|Yes (for external API/console)
-
-|External DNS
-|No
-|No
-|Yes
-|Yes (for customer-facing DNS)
-
-|NTP (Boundary)
-|No
-|No
-|Yes
-|Yes (NAT only, for reaching external NTP servers)
-
-|NTP (Internal)
-|No
-|No
-|Yes
-|No
-
-|Internal DNS
-|No
-|No
-|Yes
-|No
-
-|CockroachDB
-|No
-|No
-|Yes
-|No
-
-|Clickhouse
-|No
-|No
-|Yes
-|No
-
-|Oximeter
-|No
-|No
-|Yes
-|No
-
-|Dendrite
-|No
-|No
-|Yes
-|No
-
-|===
+```mermaid
```

Comment:
If it doesn't make it too ugly I might consider putting an explicit box of customer systems to the right of `external network`.

Reply:
Good idea.  Done.